skglm/tests/test_lbfgs_solver.py

```diff
@@ -36,5 +38,46 @@ def test_lbfgs_L2_logreg():
     )
 
 
+@pytest.mark.parametrize("use_efron", [True, False])
+def test_L2_Cox(use_efron):
+    try:
+        from lifelines import CoxPHFitter
+    except ModuleNotFoundError:
+        pytest.xfail(
+            "Testing L2 Cox Estimator requires `lifelines` packages\n"
+            "Run `pip install lifelines`"
+        )
+
+    alpha = 10.
+    n_samples, n_features = 50, 10
+    random_state = 0
```

Comment:
pass it directly in the function, it's not informative to the reader (contrary to n_samples and n_features)